lib/SILOptimizer/Transforms/ARCCodeMotion.cpp

```diff
@@ -180,6 +180,10 @@ class CodeMotionContext {
   /// we compute the genset and killset.
   llvm::SmallPtrSet<SILBasicBlock *, 8> InterestBlocks;
 
+#ifndef NDEBUG
+  Optional<SILPrintContext> printCtx;
```

Comment:
What's the purpose of printCtx? To set the SortedSIL flag?

Reply:
I added a comment:
```
+  // SILPrintContext is used to print block IDs in RPO order.
+  // It is optional so only the final insertion point interference is printed.
   Optional<SILPrintContext> printCtx;
```

It looks like I copied this from RLE's debug output.

Reply:
ok, makes sense. Thanks